common/init.sls

```diff
@@ -45,6 +45,9 @@ servo:
     - shell: /bin/bash
     - home: {{ common.servo_home }}
 
+UTC:
```

Comment:
Use Jinja to render this state conditionally, only on non-macOS, using an `{% if ... %}` block.